packages/fxa-settings/src/pages/Signin/ReportSignin/interfaces.ts

```diff
@@ -0,0 +1,8 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export interface ReportSigninProps {
```

Comment:
What is the norm for pulling interfaces into their own file? Is this better to keep it close to the calling code?

Reply:
For the recently converted pages, we've been pulling out interfaces into their own file. Probably most useful when there are multiple interfaces though, so I could go either way for this one.